index.html

```diff
@@ -23,17 +24,164 @@
         <div class="connect"><p>LET'S CONNECT</p></div>
         
         <ul class ="icons">
-            <li class="twitter"><a href="#"><img src = "./Icons/Twitter.png"></a></li>
-            <li class="linkdin"><a href="#"></ahref><img src = "./Icons/Linkedin icon.png"></a></li> 
-            <li class="m"><a href="#"></ahref><img src = "./Icons/M.png"></a></li>
-            <li class="cat"><a href="#"><img src = "./Icons/cat.png"></a></li>
-            <li class="p"><a href="#"><img src = "./Icons/Peace.png"></a></li>
+            <li class="twitter"><a href="#"><img alt="#" src = "./Icons/Twitter.png"></a></li>
+            <li class="linkdin"><a href="#"><img alt="#" src = "./Icons/Linkedin icon.png"></a></li> 
+            <li class="m"><a href="#"><img alt="#"src = "./Icons/M.png"></a></li>
+            <li class="cat"><a href="#"><img alt="#" src = "./Icons/cat.png"></a></li>
+            <li class="p"><a href="#"><img alt="#" src = "./Icons/Peace.png"></a></li>
         </ul>
             
         
         
+    </section>
+
+    <section class="middle-section">
+        <div class="tonic"><img alt="#" src ="./Work/Tonic.png"> </div>
+        <div class ="header" id ="title"><h1>Tonic</h1></div>
+        <ul class="details">
+            <li><div class ="canopy"><p>CANOPY</p></div></li>
+            <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+            <li><div class ="spec"><p>Back End Dev</p></div></li>
+            <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+            <li><div class ="spec"><p>2015</p></div></li>
```

Comment:
- [ ] Consider using a constant indentation of two spaces in all your documents. this is a convention and also best practice.